internal/pkg/agent/cmd/svc_update_linux.go

```diff
@@ -0,0 +1,48 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
```

Comment:
Should this file be moved to `internal/pkg/agent/application/upgrade/service_update_linux.go` so it lives along side `internal/pkg/agent/application/upgrade/service.go`?

Reply:
Yes.